doc/tut1.rst

```diff
@@ -641,6 +641,16 @@ initialised with the type's default value. Note that referential data types will
 be ``nil`` at the start of the procedure, and thus may require manual
 initialisation.
 
+A procedure that does not have any ``return`` statement and does not use the
+special ``result`` variable returns the value of its last expression. For example,
+this procedure
+
+.. code-block:: nim
+    :test: "nim c $1"
```

Comment:
```suggestion
```

Reply:
The majority of code samples in this file have that line, while removing it? Doesn't that verify the code is well formed?

Reply:
Exactly, please ignore @juancarlospaco this time. (No offense implied.)